mappings/net/minecraft/loot/entry/LootPoolEntry.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_79 net/minecraft/loot/entry/LootEntry
+CLASS net/minecraft/class_79 net/minecraft/loot/entry/LootPoolEntry
```

Comment:
Why this rename? To be consistent with reference in `Registry`?

Reply:
Yep